ccd-definition/CaseEvent/Camunda/CaseEvents.json

```diff
@@ -49,5 +49,16 @@
     "ShowSummary": "N",
     "ShowEventNotes": "N",
     "RetriesTimeoutURLAboutToSubmitEvent": 0
+  },
```

Comment:
Should this also not be in GA specific file?

Reply:
Good spot Smita. I am on it.